views/index.py

```diff
@@ -444,10 +444,10 @@ def add_like():
                                                        .first().likes)})
 
 
-@library.route('/deleteWish/<int:wish_id>', methods=['GET', 'POST'])
+@library.route('/delete-wish/<int:wish_id>', methods=['GET', 'POST'])
```

Comment:
As the PR mainly concern typos, could also rename other bad names like  addLike, addWish etc.

Reply:
After fixes pls ping me on slack for fast approve :)